pkgs/applications/editors/vscode/generic.nix

```
@@ -69,9 +69,9 @@ in
 
     installPhase =
       if system == "x86_64-darwin" then ''
-        mkdir -p $out/lib/vscode $out/bin
-        cp -r ./* $out/lib/vscode
-        ln -s $out/lib/vscode/Contents/Resources/app/bin/${executableName} $out/bin
+        mkdir -p "$out/Applications/${longName}.app" $out/bin
+        cp -r ./* "$out/Applications/${longName}.app"
+        ln -s "$out/Applications/${longName}.app/Contents/Resources/app/bin/code" $out/bin/${executableName}
```

```suggestion
        ln -s "$out/Applications/${longName}.app/Contents/Resources/app/bin/${executableName}" $out/bin/${executableName}
```

Why was that ^ changed? Is that not important for the macos build artifact?

---

This was done because for whatever reason the vscodium macOS release .zip doesn't have the `code` utility renamed to `codium`

---

I see, can you add a comment? This installPhase is already completely uncommented, and if I were to notice this not using `executableName` I would for sure change that and break things.

---

I would be surprised to see a PR that changes working code from a static path reference to using a variable. I agree that a comment is probably a good idea, but my first impression upon reading this code is simply that the app includes a binary named `app/bin/code`, and I wouldn't question that.